hack/check-docforge.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+echo "> Installing docforge"
+
+TOOLS_BIN_DIR=${TOOLS_BIN_DIR:-$(dirname $0)/tools/bin}
+
+platform=$(uname -s | tr '[:upper:]' '[:lower:]')
+version="0.21.0"
+file_name="docforge-${platform}-amd64"
+
+temp_dir="$(mktemp -d)"
+function cleanup {
+  rm -rf "${temp_dir}"
+}
+trap cleanup EXIT ERR INT TERM
+
+curl -L -o ${temp_dir}/${file_name} "https://github.com/gardener/docforge/releases/download/v${version}/${file_name}"
```

Comment:
This downloads the binary in every text execution, right?

@timebertt Is this the correct way how to deal with tools?

Reply:
Maybe I can add one If clause which downloads the binary only if it does not exist in the tools dir?

Reply:
Sounds reasonable, yes

Reply:
The PR is updated